p2p/src/lib.rs

```diff
@@ -315,6 +316,37 @@ pub fn make_p2p<S: PeerDbStorage + 'static>(
     time_getter: TimeGetter,
     peerdb_storage: S,
 ) -> Result<P2pInit<S>> {
+    match chain_config.chain_type() {
+        ChainType::Mainnet | ChainType::Testnet | ChainType::Signet => {
+            // These are protocol-sensitive parameters and changing them has to be done with care.
+            // If the change has been sufficiently scrutinized an is deemed desirable, the values
+            // should be updated here as well.
+            //
+            // What to consider when changing these:
+            // * The `max_clock_diff` and `handshake_timeout` settings are related. The effective
+            //   max allowed difference between our clock and peer's clock is the sum of the two.
+            //   If one is changed, it should be considered how the other should be affected.
+            // * Peer may be stalled for the duration of up to `effective_max_clock_diff`.
+            //   This may interact with other timeouts.
```

Comment:
@TheQuantumPhysicist Anything to be added to the comment?

Reply:
I would just add that the implementation of handshake tolerance is in what struct/function as a soft reference to understand how this works. Otherwise, this looks good.